app/v2/connectors/PersistenceConnector.scala

```diff
@@ -195,4 +203,17 @@ class PersistenceConnectorImpl @Inject() (httpClientV2: HttpClientV2, appConfig:
       )
       .addParam("movementEORI", movementEORI.map(_.value))
 
+  def patchMessage(movementId: MovementId, messageId: MessageId, body: MessageUpdate)(implicit
+    hc: HeaderCarrier,
+    ec: ExecutionContext
+  ): Future[Unit] = {
+    val url = appConfig.movementsUrl.withPath(updateMessageRoute(movementId, messageId))
+
+    httpClientV2
+      .patch(url"$url")
+      .withBody(Json.stringify(Json.toJson(body)))
+      .setHeader(HeaderNames.ACCEPT -> MimeTypes.JSON)
```

Comment:
I believe here we need to set the Content-Type header  ``` HeaderNames.CONTENT_TYPE, equalTo(MimeTypes.JSON) ``` instead of Accept header?

Reply:
Yes you're right, we're sending json but not expecting any content back